### PR TITLE
Extended Datetime Cfgu Types

### LINKED ISSUE #373: Extend `Date Time` Cfgu Types
### Suggestion

- Durations: 1h, 2d, etc..
- date formats etc..

### Motivation

_No response_

### Context

_No response_

### PR DESCRIPTION
**Pull request by JayGaba**
# Closes #373 
1. Duration:

- Added a regex pattern: ^\d+[smhdwMy]$
   -  Description: Validates durations like 1h, 2d, 3w.

2. DateFormat:
- Added a regex pattern: ^(yyyy|yy|MM|M|dd|d|HH|H|mm|m|ss|s)([.-/\s]?(yyyy|yy|MM|M|dd|d|HH|H|mm|m|ss|s))*$
    - Description: Ensures date formats like yyyy-MM-dd or dd/MM/yyyy HH:mm:ss are valid

### CODE REVIEW

File: packages/schema/.cfgu.json
Comment (RichardAkman) [inline packages/schema/.cfgu.json:1]: Please update your PR to account for the conflicts with your other PR that we merged 🙏🏻 

Also please remove the type validation from this file, validation of types is the job of `EvalCommand`

**Review by RichardAkman — CHANGES_REQUESTED** [review]
Please update your PR to account for the conflicts with your other PR that we merged 🙏🏻

Also please remove the type validation from this file, validation of types is the job of EvalCommand

**Review by RichardAkman — APPROVED** [review]
LGTM, thank you for your contributions ⭐ ⭐

**Comment by pelegpor** [comment]
Hi @JayGaba ,

Here are the details for claiming your Hacktoberfest prizes! 🎉 Check out [#465  (comment)](https://github.com/configu/configu/issues/465#issuecomment-2498550259) for more information or simply fill out the form here: [Claim Your Swag](https://share-eu1.hsforms.com/2SVc5VKY3Thi04Bvjh17WKw2e62o0).  

Remember, you only need to fill out the form once for all your PRs. 😊

**Pull request merged**